domains/cherry.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "hhchenhhhhhhh",
+        "email": "hkdkfih@gmail.com"
+    },
+    "record": {
+        "URL": "cherryservice.my.to"
```

Comment:
1. Please use a valid record.
2. This domain cannot be registered due to abuse risks.